### ISSUE TITLE
aria-hidden=1 breaks screen reader accessibility

### ISSUE THREAD

**Issue by slyon** [canonical-web-and-design/netplan.io#176]
## Summary

I got a bug report via private email recently, which I'd like to forward to the web team:

"I am blind, and I was trying to read the netplan.io docs off your website.
Unfortunately, the body element of the page has an aria-hidden=1 attribute on the html element.
So, Orca with Firefox ignores the entire DOM when you try to access the website with a screen reader.
Norally, I'd try to find a way to report this on the website, but it's kind of hard to do since I cannot read the website at all."

## Process

* Setup Orca screen reader
* Access netplan.io website

## Current and expected result

A screen reader should be able to read the netplan.io website.

**Comment by SirSamTumless** [comment]
@sowasred2012 Do you have time to investigate this.

**Comment by hartmans** [comment]
I notice the issue is closed.  Should I now see the netplan.io website using Orca in Firefox? Or is there some period it will take to propagate?

**Comment by sowasred2012** [comment]
Hey @hartmans @slyon - I'm in the process of releasing it at the moment, it shouldn't take too long but I'll update here when it has.

Once this is resolved, I'd be interested to know if there other glaring accessibility issues we can address ASAP.

**Comment by sowasred2012** [comment]
@hartmans @slyon the changes are live now, you may need to clear your browser's cache. I've tested it with Orca and Chrome's screen reader, but please let me know if you still see the issue.

**Comment by hartmans** [comment]
It looks like it works well enough that I can read the docs online,
which is what I cared about!
Thanks!
